Airline_Web_API/Airline_Web_API/Startup.cs

```diff
@@ -48,6 +48,10 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddAutoMapper(AppDomain.CurrentDomain.GetAssemblies());
 
+            services.AddControllers().AddNewtonsoftJson(options =>
+                    options.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore
+                );
```

Comment:
Indent

Reply:
Also why do you expect loops in your json? Maybe you should design your models in such way that won't be possible?